CHANGES.rst

```diff
@@ -10,7 +10,9 @@ Unreleased
 
 - :meth:`.pin` to manage pinned submissions on the authenticated user's profile.
 - :meth:`.update_display_layout` to update the display layout of posts in a
-  :class:`.Collection`.
+  :class:`.CollectionModeration`.
```

Comment:
```suggestion
  :class:`.Collection`.
```
This needs to be ``:class:`.Collection` ``.

Reply:
Why? That PRAW function exists inside class CollectionModeration.

Reply:
It's for collections though. You call it with `collection.mod.update_display_layout()`. You have to have a `Collection` first.

Reply:
Ah, ok. I'll use that logic next time. I thought the method is supposed to refer back to the class it exists in.